doc/sphinx/source/frameworks/generic-configs.rst

```diff
@@ -20,15 +20,45 @@ something like:
 
 
 Find or display nGraph Version
--------------------------------
-
+==============================
 
+If you're working with the :doc:`../python_api/index`, the following command 
+may be useful:
 
 .. code-block:: console
 
    python3 -c "import ngraph as ng; print('nGraph version: ',ng.__version__)";
 
 
+Localize environment variables
```

Comment:
The word `Localize` makes me think about language localization and internationalization. Can we use a different word?

Reply:
![more-genconfig](https://user-images.githubusercontent.com/50558/60282721-7a9e9e80-98bc-11e9-9bb7-a3d695e36221.png)